Library/Homebrew/extend/os/linux/formula.rb

```diff
@@ -19,7 +19,7 @@ def allowed_missing_lib?(lib)
       when Regexp
         x.match? lib
       when String
-        lib.include? x
+        lib.to_s.include? x
```

Comment:
Why is `to_s` necessary when it's already a `String`? How about an `else ... odie` or making `when String` be `else` instead?

Reply:
It's `x` which is a String here and `lib` is a `String` when this function is called from `linkage_checker`. But it can be anything in `brew irb`. So, this `to_s` is mainly to handle things like
```
brew irb <<< ":libxcb.f.allowed_missing_lib? 1
```
I guess I should check the type of `lib` instead. I'll do that.

Reply:
Yeh or you could just do `if Regexp` and otherwise rely on doing `.to_s` on either/both.